### PR TITLE
Add English semantic search of pipelines

### PR DESCRIPTION
**Pull request by w5688414**
### PR types

+ New features

### PR changes

+ Models&APIs

### Description

+ Add English semantic search of pipelines with rocketqav2

### CODE REVIEW

File: paddlenlp/transformers/semantic_search/modeling.py [hunk 1]
```diff
@@ -96,6 +97,15 @@ def __init__(self,
         assert (self.query_ernie is not None) or (self.title_ernie is not None), \
             "At least one of query_ernie and title_ernie should not be None"
 
+        # Compatible to rocketv2 initialization
+        if (reinitialize):
```
Comment (wawltor) [inline paddlenlp/transformers/semantic_search/modeling.py]: 这块的语法有点奇怪，不需要加括号吧

Comment (w5688414) [reply paddlenlp/transformers/semantic_search/modeling.py]: 已修改

**Review by wawltor — DISMISSED** [review]
LGTM

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**